terraform/dms/security_groups.tf

```diff
@@ -5,30 +5,22 @@ data "aws_security_groups" "rds_broker_db_clients" {
   }
 }
 
-resource "aws_security_group" "secrets_manager_dms_access" {
-  name        = "${var.env}-secrets-manager-dms"
+resource "aws_security_group" "dms" {
+  name        = "${var.env}-dms"
   description = "Allow HTTPS inbound traffic for security manager vpc endpoint"
```

Comment:
Description doesn't really match the name

Reply:
Fixed